docs/source/tutorials_source/package/tutorial_custom_augmentations.py

```diff
@@ -0,0 +1,244 @@
+"""
+.. _lightly-custom-augmentation-5:
+
+Tutorial 5: Custom Augmentations
+==============================================
+
+"""
+
+# %%
+# Imports
+# -------
+#
+# Import the Python frameworks we need for this tutorial.
+import os
+import glob
+import torch
+import torch.nn as nn
+import torchvision
+import pytorch_lightning as pl
+import lightly
+import matplotlib.pyplot as plt
+from sklearn.neighbors import NearestNeighbors
+from sklearn.preprocessing import normalize
+from PIL import Image, ImageOps
+import numpy as np
+import pandas
+
+# %%
+# Configuration
+# -------------
+# TODO
+#
+num_workers = 8
+batch_size = 64
+num_splits = 0
+seed = 1
+max_epochs = 100
+num_ftrs = 500
+
+# %%
+# Let's set the seed for our experiments
+pl.seed_everything(seed)
+
+# %%
+# Make sure `path_to_data` points to the downloaded x-ray dataset.
+# You can download the dataset `from kaggle <https://www.kaggle.com/c/vinbigdata-chest-xray-abnormalities-detection/overview>`_.
```

Comment:
Images in this dataset are in DICOM format which is great. I would also use DICOM in this tutorial since it's a very common format for medical imaging. To convert from DICOM to NumPy/torch we can use this library: https://github.com/pydicom/pydicom